scripts/multi_ldap_change.php

```diff
@@ -0,0 +1,135 @@
+<?php
+#==============================================================================
+# Includes
+#==============================================================================
+require_once(__DIR__."/../conf/config.inc.php");
+require_once(__DIR__."/../lib/functions.inc.php");
+
+#==============================================================================
+# Action
+#==============================================================================
+
+$result = "";
+$return = Array();
+$error_code = 1;
+$login = $argv[1];
+$newpassword = $argv[2];
+$oldpassword = '';
+
+$log_file = fopen(__DIR__.'/multi_ldap_change.log', 'a+');
+fwrite($log_file, "Change '$login' password...\n");
```

Comment:
We should not log in scripts/ directory by default.

Use instead https://www.php.net/manual/fr/function.sys-get-temp-dir.php